components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/EndpointRegistry.java

```diff
@@ -92,7 +98,8 @@ EndpointRegistryInfo getEndpointRegistryByUUID(String registryId, String tenantD
 
     /**
      * Updates an existing endpoint registry
-     * @param registryId uuid of the endpoint registry which needs to be updated
+     *
+     * @param registryId           uuid of the endpoint registry which needs to be updated
```

Comment:
Seems some formatting issue

Reply:
No. It is aligned with the next param @param endpointRegistryInfo. large gap is because the second param is long